### PR TITLE
Doc api + minor api fixes

### PR DESCRIPTION
**Pull request by buixor**
- add a dedicated doc on api usage
 - rename developpers tab to swagger
 - link user's guide on metabase without docker

### CODE REVIEW

File: docs/v1.X/docs/localAPI/howto.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+```bash
+▶ curl  -H "X-Api-Key: 837be58e22a28738066de1be8f53636b"  http://localhost:8080/v1/decisions\?scope\=username\&value\=myuser
+[{"duration":"3h57m59.021170481s","id":2338,"origin":"cscli","scenario":"manual 'enforce_mfa' from '939972095cf1459c8b22cc608eff85daEb4yoi2wiTD7Y3fA'","scope":"username","type":"enforce_mfa","value":"myuser"}]
+```
+
+> Query a given decision type/user
+
+```bash
+▶ curl  -H "X-Api-Key: 837be58e22a28738066de1be8f53636b"  http://localhost:8080/v1/decisions\?scope\=username\&value\=myuser
```
Comment (erenJag) [inline docs/v1.X/docs/localAPI/howto.md:165]: query params should be updated to type/user

File: docs/v1.X/docs/localAPI/howto.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
+
+
+!!! info
+
+    This page explains how to interract with the local API exposed by crowdsec.
```
Comment (AlteredCoder) [inline docs/v1.X/docs/localAPI/howto.md:5]: interact*

File: docs/v1.X/docs/localAPI/howto.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+    It's meant to be useful for system administrators, or users that want to create they own bouncers.
```
Comment (AlteredCoder) [inline docs/v1.X/docs/localAPI/howto.md:7]: their* ?

**Review by registergoofy — APPROVED** [review]
lgtm

**Pull request merged**